examples/bpk-component-bottom-sheet/examples.tsx

```diff
@@ -0,0 +1,195 @@
+    maximus erat. Morbi sed eros vitae augue accumsan dictum sit amet eu lectus.
+    Integer vitae consectetur libero, sed porttitor urna.
+  </Paragraph>,
+  <Paragraph>
+    In arcu leo, bibendum in vestibulum quis, vulputate sed nisl. Donec sit amet
+    turpis quis metus viverra rutrum id id elit. Duis luctus, mauris ut accumsan
+    vehicula, magna lorem posuere velit, nec laoreet magna ante ut nulla.
+    Vivamus vestibulum bibendum purus quis dictum. In accumsan convallis tempor.
+    Proin porta massa et metus viverra feugiat. Aenean blandit pellentesque
+    nunc, in interdum magna molestie non. Suspendisse pretium lectus et libero
+    fringilla placerat. Aliquam pellentesque odio non maximus egestas. Nam
+    feugiat mi ac neque sodales, in euismod dolor varius.
+  </Paragraph>,
+  <Paragraph>
+    Aenean tempus tempus lorem in consequat. Quisque nec felis vitae mi commodo
+    ultricies sit amet in lectus. Praesent imperdiet auctor nisl sit amet
+    ultricies. Donec posuere placerat nulla a scelerisque. Nulla sit amet
+    eleifend magna. Ut eu cursus metus, id pulvinar lectus. Proin euismod sed ex
+    vel dignissim. Donec faucibus nec risus eu pellentesque. Cras vulputate
+    varius volutpat. Duis ut nisi nulla. Duis volutpat lectus purus. Aliquam
+    placerat dignissim mauris vitae interdum. Donec venenatis mattis mi eu
+    facilisis. Maecenas pellentesque eros elementum, tincidunt tortor ac,
+    fringilla massa. Cras sed orci nec erat porttitor lacinia vitae sed arcu.
+  </Paragraph>,
+  <Paragraph>
+    Nunc lobortis arcu eget tellus tincidunt commodo. Phasellus ac tortor
+    turpis. Cras ac quam non metus iaculis sollicitudin. Vestibulum ante ipsum
+    primis in faucibus orci luctus et ultrices posuere cubilia Curae; Morbi
+    aliquam consectetur nisl at fermentum. Phasellus nisi arcu, fermentum ut
+    malesuada eu, ultrices nec enim. Donec eleifend eros mauris, eu rutrum
+    tellus suscipit ac. Pellentesque finibus mollis arcu, non tempor lorem
+    gravida at. Nam laoreet neque quis gravida blandit. Mauris pharetra urna
+    hendrerit pretium auctor. Aliquam erat volutpat. Class aptent taciti
+    sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+    Fusce quis felis non lectus egestas pretium id sed erat. Aliquam dapibus
+    erat sit amet facilisis luctus.
+  </Paragraph>,
+  <Paragraph>
+    Vestibulum convallis ut nulla in iaculis. Aliquam erat volutpat. Nullam non
+    semper sem. Ut gravida est eu nisi condimentum, lobortis gravida ipsum
+    tincidunt. Duis lacinia tincidunt suscipit. Maecenas tincidunt quam ipsum,
+    non sodales ante placerat in. Suspendisse malesuada auctor erat, vel
+    pulvinar erat dignissim vitae.
+  </Paragraph>,
+];
+
+class BottomSheetContainer extends Component<
+  {
+    children: ReactNode,
+  },
+  {
+    isOpen: boolean,
+  }
+> {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
```

Comment:
As we use TS we don't use `PropTypes` and definitions like this in our components anymore.

So as you have a definition in the constructor you should not need this here :)